sagemaker-debugger/model_specific_realtime_analysis/bert_attention_head_view/utils/attention_head_view.py

```diff
@@ -0,0 +1,92 @@
+from bokeh.plotting import show, figure
+from bokeh.models.annotations import Title
+from bokeh.models import  ColumnDataSource, Label, Range1d
+from bokeh.io import  show, output_notebook, push_notebook
+from bokeh.models.glyphs import Line
+import numpy as np
+
+output_notebook()
+
+class AttentionHeadView():
+    def __init__(self, 
+                 input_tokens=None, 
+                 tensors=None, 
+                 layer='bertencoder0_transformer0_multiheadattentioncell0_output_1', 
```

Comment:
[nit pick]: While is makes sense to keep default value here as a backup, I think it would help to also pass it as parameter in corresponding cells in notebook itself - just to signify that layer name is configurable.

Reply:
I added the argument to corresponding cell in the notebook.